README.md

```diff
@@ -69,7 +69,7 @@ Manually create a mysql container in openshift, make note of the db name, userna
 Remove all objects created during tests : 
 ```bash
 # Assuming all the names for items created contain 'pied'
```

Comment:
No. Nohotdog.

pied is the first part of the names in the example json, so matches them all.